### PR TITLE
foreign key and free text fields are now case insensitive by default,…

### PR DESCRIPTION
**Pull request by fredkingham**
… refs #1390 

Note this doesn't contain a migration to fix this as I think this should probably be done at the application level.

### CODE REVIEW

File: doc/docs/reference/core_fields.md [hunk 1]
```diff
@@ -32,3 +32,6 @@ enum('one', '2', 'III')
 
 A field that stores it's value as a generic foreign key to an Opal LookupList
 or as the value in a CharField.
+
+By default this is case insensitive, pass in `case_sensitive=True` to make
```
Comment (davidmiller) [inline doc/docs/reference/core_fields.md]: I'd like a code example below this and more of an explanation of what 'case [in]sensitive' means in this context?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 ### 0.11.0 (Major Release)
 
+#### Changes all models
```
Comment (davidmiller) [inline CHANGELOG.md]: I think you've committed the changelog with the title stuff in as well - I deleted that blank 'changes all models' heading and moved the title section in a previous PR...

**Comment by davidmiller** [comment]
The reason we don't need a migration is that applications will continue to retrieve FT values which are differently cased versions of lookuplist entries, but stop saving new ones ?

I think we might want a bigger 'warning this upgrade will change the behaviour of your existing models with lookuplists' section in the upgrade guide.

File: opal/tests/test_core_fields.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 """
 Unittests for the opal.core.fields module
```
Comment (davidmiller) [inline opal/tests/test_core_fields.py]: You know, I think we may not have any explicit unittests for FT in this file, which I would really expect !

Comment (fredkingham) [reply opal/tests/test_core_fields.py]: I tested but the True and False cases in the methods


ie This is the case sensitive miss for a lookup list
https://github.com/openhealthcare/opal/pull/1534/files/96b2cfad409cc07583ceecdab4ed63be73371110#diff-7984144a820e90113b1c914866eb396aR117

This is a case sensitive miss for a synonym
https://github.com/openhealthcare/opal/pull/1534/files/96b2cfad409cc07583ceecdab4ed63be73371110#diff-7984144a820e90113b1c914866eb396aR143

**Comment by fredkingham** [comment]
changing this to merge with v0.12.0

**Pull request closed without merging**